projects/synthetic_brca/synthetic_brca_load.py

```diff
@@ -0,0 +1,43 @@
+import root_directory
+import json
+from src.doccano_utils import DoccanoSession
+
+########################## DEFINE SET-UP VARIABLES ##########################
+PROJECT_DETAILS = {
+    'name': 'Testing Class: Synthetic BRCA Classification',
+    'description': 'Classification of synthetic BRCA testing reports',
+    'project_type': 'DocumentClassification',
+    'guideline': 'Classify BRCA results under the most appropriate label(s)'
+}
+LABELS = ['BRCA1 positive', 'BRCA2 positive', 'BRCA1 VUS', 'BRCA2 VUS', 'Invalid']
+DATA_FILE = 'data/brca_reports.json'
+#############################################################################
+
+def main():
+    # connect and log on to doccano
+    session = DoccanoSession()
+    print(f"Connected to Doccano as user: {session.username}")
+
+    # create project
+    project = session.create_project(**PROJECT_DETAILS)
+    print(f"Created new project: {project.name}, {project.id}")
+
+    # set up labels
+    session.setup_labels(LABELS)
+    print(f"Created {len(LABELS)} labels")
+
+    # load json from data file
+    try:
+        with open(DATA_FILE, 'r') as file:
+            data = json.load(file)
+    except Exception as e:
+        print(f"Failed to load samples: {str(e)}")
+        return
```

Comment:
Same thing as my other comment elsewhere, could do this if you want?

```suggestion
    try:
        with open(DATA_FILE, 'r') as file:
            data = json.load(file)
    except Exception as e:
        print(f"Failed to load samples.")
        raise e
```